code/modules/hydroponics/plant_genes.dm

```diff
@@ -280,6 +280,15 @@
 	..()
 	G.set_light(glow_range(G.seed), glow_power(G.seed), glow_color)
 
+/datum/plant_gene/trait/glow/shadow
+	//makes plant emit slightly purple shadows
+	//adds -potency*(rate*0.2) light power to products
+	name = "Shadow Emission"
+	rate = 0.04
+
+/datum/plant_gene/trait/glow/shadow/glow_power(obj/item/seeds/S)
+	return -max(S.potency*(rate*0.2), 0.2)
```

Comment:
I'm not sure it's precisely darkness; it might just be visually inverted light color, which isn't the same and probably doesn't cause light to react the same